### ISSUE TITLE
Default working directory for PowerShell should be the user's directory

### ISSUE THREAD

**Issue by hach-que** [PowerShell/PowerShell#7376]
Steps to reproduce
------------------

Open PowerShell using the Start Menu link on Windows.

Expected behavior
-----------------

The default prompt should be located at `$env:USERPROFILE` or `$env:USERPROFILE\Documents` on Windows.

Actual behavior
---------------

The default prompt opens at `C:\Program Files\PowerShell\6.0.2`.

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.2
PSEdition                      Core
GitCommitId                    v6.0.2
OS                             Microsoft Windows 10.0.17134
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

Why this should be changed
----------------

The default for PowerShell 5.x is $env:USERPROFILE, as outlined in this screenshot:

![image](https://user-images.githubusercontent.com/504826/43293812-72b8f75a-917f-11e8-8016-c1cc50b6cb2f.png)

This is both versions of PowerShell, with PowerShell Core just being the default installation from the provided MSI.

Every time I want to use PowerShell Core, I have to `cd` to my user profile directory before I can get any work done, since the default working directory isn't writable (and even if it was, I wouldn't really want to be writing things into there). In addition, since it's "so far away" from the directories I normally operate in, I have to `cd` with an absolute path which means typing a long directory path correctly every time.

If this was something I had to do once then it wouldn't be so bad, but it's *every time* I open a PSCore prompt on Windows, so it gets tiring after a while.

**Comment by rkeithhill** [comment]
This is a dupe of #5705 which has been fixed in `6.1-preview4`.

**Comment by SteveL-MSFT** [comment]
PSCore6.1 is nearing GA so it doesn't make sense to spend the effort to backport the fix to 6.0.x

**Comment by hach-que** [comment]
Thanks! Looking forward to 6.1 :)
